TestRunner.psm1

```diff
@@ -479,103 +662,125 @@ function Duplicate-ChangeVersionTracker {
     Copy-Item 'change_version_tracker.json' -Destination $testResultsPath
 }
 
-function Get-CredentialOrDefault($server) {
-    if ($server -eq 'localhost') {
+function Get-CredentialOrDefault {
+    param (
+        [Parameter(Mandatory=$True)]
+        [string]
+        $Server
+    )
+
+    if (Test-IsLocalhost $Server) {
+        throw "This shouldn't happen" + (Get-PSCallStack)
         return $null
     } else {
-        # First, attempt to load credentials previously registered with Register-Credentials.
-        # If there is no such registered credential, asks the user for credentials interactively.
+        $credentials = Get-StoredCredential -Target $server
```

Comment:
A missing method? Seeing error 
Get-CredentialOrDefault : The term 'Get-StoredCredential' is not recognized as the name of a cmdlet, function, script
file, or operable program. Check the spelling of the name, or if a path was included, verify that the path is correct
and try again.

Reply:
You'll need to install the credential manager module, see updated user-guide.md